CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ find_package(KF5 COMPONENTS
 
 find_package(KF5Archive 5.85.0 CONFIG)
 find_package(KF5Auth CONFIG)
+find_package(KF5SyntaxHighlighting CONFIG)
```

Comment:
adding the note here: please improve the commit message, at the very least give a summary of how you implement the syntax highlighting